topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.md

```diff
@@ -0,0 +1,284 @@
+
+Data can access in several ways.  
+
+1. Files can be imported from the computer 
+2. Using direct link. 
+
+
+### Get data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]() or from the shared data library
+>
+>    ```
+>    https://zenodo.org/record/4111092/files/ACPs.fasta
+>    https://zenodo.org/record/4111092/files/non_ACPs.fasta
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+> 3. Rename the datasets to their basename (ACPs.fasta, non_ACPs.fasta)
+> 4. Check that the datatype is correctly set to fasta
+>
+>    {% include snippets/change_datatype.md datatype="fasta" %}
+>
+>
+{: .hands_on}
+
+
+
+## Background
+
+Biological molecules such as proteins, peptides, DNA, and RNA can be represented by their biochemical or sequences-based properties. These properties can be utilized to deduce biological meanings. Properties associated with a peptide sequence such as overall charge, hydrophobicity profile, or k-mer composition can be utilized to build a machine learning model and predict the biological properties of unknown peptides. Finding anticancer peptides (ACPs) through wet-lab methods is costly and time-consuming; thus, the development of an efficient computational approach is useful to predict potential ACP peptides before wet-lab experimentation.
+In this tutorial, we used 6 different machine learning algorithms for the prediction of ACPs using the peptide sequence-based CTD (Composition transition and distribution descriptors) descriptors or features. All the models were trained using our dataset that combines 138 Anticancer peptides and 138 non-anticancer peptides. We applied 10 fold cross-validation on this data set.
+
+
+## Calculating Peptide Descriptors 
+
+A descriptor or feature is the quantitative or a qualitative measure of a property that is associated with a sequence. For example, a chemical compound can be described via its charge chemical formula, molecular weight, number of rotatable bonds, etc. Similarly, several properties can be deduced from the biological sequence that can be utilized to summarise a biological property such as anti-cancer activity. In this example, we utilized CTD descriptors, describes as composition, transition, and distribution descriptors. 
+
+![Alternative text](../../images/PDAUG_ML_1.png "ML algorithms use numerical representation of a sequence-based properties for model building")
+
+### Calculating descriptor for ACPs
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. **PDAUG Sequence Property Based Descriptors** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input fasta file"*: `ACPs.fasta` (output of **Input dataset** {% icon tool %})
+>    - *"DesType"*: `CTD`
+>
+>
+{: .hands_on}
+
+### Calculating descriptor for non-ACPs
+
```

Comment:
Please merge multiple hands-on boxes that are next to one another

Reply:
Merged